docs/core/linux-prerequisites.md

```diff
@@ -22,25 +22,26 @@ This article shows the dependencies needed to develop .NET Core applications on
 
 .NET Core 2.x is supported on the following Linux distributions/versions:
 
- * Red Hat Enterprise Linux 7 x64
- * CentOS 7 x64
- * Oracle Linux 7 x64
- * Fedora 25, 26 x64
- * Debian 9, 8.7+ x64 
- * Ubuntu 17.04, 16.04, 14.04  x64
- * Linux Mint 18, 17 x64
- * openSUSE 42.2+ x64
- * SUSE Enterprise Linux (SLES) 12 SP2+ x64
+ * Red Hat Enterprise Linux 7 x86_64
```

Comment:
While I think this is proper from a RHEL arc identification perspective, .NET Core will not run on x86 Linux.

Reply:
We'd like to list the distros as 64-bit (x86_64)? 
Example:
•	Red Hat Enterprise Linux / CentOS / Oracle Linux 7 64-bit (x86_64) 

 @omajid - thoughts?

Reply:
Works for me. Would you like me to update the patch?

Reply:
There have been significant changes in that article since you opened this.  It might be easier to close this PR and open a new one and add the changes.

Reply:
And thank you, @omajid, for your feedback and assistance! It has been a great help.

Reply:
No problem. Glad to help.

Closing this in favour of https://github.com/dotnet/docs/pull/3085